staging/src/k8s.io/legacy-cloud-providers/azure/azure_wrap.go

```diff
@@ -198,6 +205,11 @@ func (az *Cloud) getSecurityGroup(crt cacheReadType) (nsg network.SecurityGroup,
 	return *(securityGroup.(*network.SecurityGroup)), nil
 }
 
+type vmEntry struct {
+	vm         *compute.VirtualMachine
+	lastUpdate time.Time
+}
+
 func (az *Cloud) newVMCache() (*timedCache, error) {
 	getter := func(key string) (interface{}, error) {
```

Comment:
note: `key` would not be used anymore with List, so we need to define a const for cache getter in L96, so that it won't confuse.

Reply:
sure, defined const.